.travis.yml

```diff
@@ -17,6 +17,6 @@ install:
   - pip install Flask-Testing
   - pip install blinker
 script:
-  - nosetests --with-cov --cov-config .coveragerc
+  - nosetests --nocapture --with-cov --cov-config .coveragerc
```

Comment:
Is this still necessary?

Reply:
There's no harm in keeping it. All it does it, not hide stdout. Very helpful while debugging test cases on CI.